bears/general/LineCountBear.py

```diff
@@ -39,3 +49,12 @@ def run(self, filename, file, max_lines_per_file: int,
                                  extra=file_length-max_lines_per_file)),
                 severity=RESULT_SEVERITY.NORMAL,
                 file=filename)
+
+        elif file_length < min_lines_per_file:
+            yield Result.from_values(
+                origin=self,
+                message=('This file has {} lines, which is {} lines fewer '
+                         'than required.'
+                         .format(file_length,
+                                 min_lines_per_file - file_length)),
```

Comment:
maybe instead of saying "this is 3 lines less than required", say "x lines are required". That makes more results from the same kind better to read, as this number won't change.